practice-project-api/routes/calculateRoutes.ts

```diff
@@ -0,0 +1,34 @@
+import { Router, Request, Response } from "express";
+
+export const calculateRoute = Router();
+
+calculateRoute.post("/calculate", (req: Request, res: Response, next) => {
+
+  type calculatorOperation = '*' | '/' | '+' | '-';
+
+  interface Calculate {
+    firstNumber: number,
+    secondNumber: number,
+    operation: calculatorOperation,
+  };
+
+  const input: Calculate = {
+    firstNumber: req.body.firstNumber,
+    secondNumber: req.body.firstNumber,
+    operation: req.body.operation,
+  };
+
+  let calculate = (firstNumber: number, operation: calculatorOperation, secondNumber: number): number => {
+    switch (operation) {
+      case '*': return firstNumber * secondNumber;
+      case '/': return firstNumber / secondNumber;
+      case '+': return (+firstNumber) + (+secondNumber);
```

Comment:
the `+` in front of a variable is redundant here. Typescript strictly types these variables as numbers so there's no need to type cast.

Reply:
I couldn't get it to work without that, and I'm not really sure why. It would just concatenate instead of add. Seems like it was treating firstValue as a string for some reason.

Reply:
Nah you shouldn't have to do this since the varaibles are strictly typed. I don't think this works atm anyway running locally.